cmd/sops/main.go

```diff
@@ -796,6 +800,7 @@ func main() {
 					KeyServices:     svcs,
 					DecryptionOrder: order,
 					IgnoreMAC:       c.Bool("ignore-mac"),
+					UseAwsProfile:   c.String("aws-profile"),
```

Comment:
There are four instances of `decryptOpts` creations in this file. You need to change all of them, not just one.